project/AkkaBuild.scala

```diff
@@ -42,40 +42,80 @@ object AkkaBuild extends Build {
     UnidocRoot.akkaSettings ++
     Protobuf.settings ++ Seq(
       parallelExecution in GlobalScope := System.getProperty("akka.parallelExecution", parallelExecutionByDefault.toString).toBoolean,
-      Dist.distExclude := Seq(actorTests.id, docs.id, samples.id, osgi.id),
+      Dist.distExclude := Seq(
+        actorTests.id,
+        benchJmh.id,
+        docs.id,
+        httpTests.id,
+        persistenceShared.id,
+        remoteTests.id,
+        samples.id,
+        streamTests.id,
+        streamTestsTck.id,
+        osgi.id
```

Comment:
it would have been nice to derive this from the `projects` filtered by the `dontPublishSettings`, but I don't know how difficult that would be